bootstrap.py

```diff
@@ -465,6 +470,25 @@ def apply_configuration(node_set, configuration_file, global_patches):
             "policyAuditMode=true",  # Audit mode, do not block traffic
         ]
 
+    # Normally Envoy has SYS_ADMIN, but that can be replaced with PERFMON and BPF, see
+    # https://github.com/cilium/cilium/blob/v1.16.1/install/kubernetes/cilium/values.yaml#L2263-L2271
+    envoy_caps = ["NET_ADMIN", "PERFMON", "BPF"]
+    if gw_api := config["cluster"]["cilium"].get("gateway-api"):
+        if gw_api["enabled"]:
+            cilium_opts += ["gatewayAPI.enabled=true"]
+            if gw_api.get("host-network"):
+                cilium_opts += ["gatewayAPI.hostNetwork.enabled=true"]
+            if gw_api.get("privileged-ports"):
+                # https://docs.cilium.io/en/stable/network/servicemesh/gateway-api/gateway-api/#bind-to-privileged-port
+                cilium_opts += [
+                    "envoy.securityContext.capabilities.keepCapNetBindService=true"
+                ]
+                envoy_caps += ["NET_BIND_SERVICE"]
+
+    cilium_opts += [
+        "envoy.securityContext.capabilities.envoy={%s}" % (",".join(envoy_caps),)
```

Comment:
Redundant comma here as well

Reply:
Same as above, but I'll refactor this one out as well.